internal/highway/crud_registry.go

```diff
@@ -1,57 +1,122 @@
 package highway
 
 import (
-	context "context"
-	"fmt"
+	"context"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
 	rtv1 "github.com/sonr-io/sonr/x/registry/types"
 	rt "go.buf.build/grpc/go/sonr-io/blockchain/registry"
 )
 
+// ParamsHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is[GET]
+func (s *HighwayServer) ParamsHTTP(c *gin.Context) {
+	params, err := s.cosmos.QueryParams()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	c.JSON(http.StatusOK, params)
+}
+
+// WhoIsHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is[GET]
+func (s *HighwayServer) WhoIsHTTP(c *gin.Context) {
+	names, err := s.cosmos.QueryAllNames()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	c.JSON(http.StatusOK, names)
+}
```

Comment:
This should not be exposed over http, we do not wish to allow the user to do an exhaustive query of this data model.